charts/evm-bridge-withdrawer/templates/configmaps.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: evm-bridge-withdrawer-env
+  namespace: {{ include "evm-bridge-withdrawer.namespace" . }}
+data:
+  ASTRIA_BRIDGE_WITHDRAWER_LOG: "astria_bridge_withdrawer=debug"
+  ASTRIA_BRIDGE_WITHDRAWER_API_ADDR: "0.0.0.0:{{ .Values.ports.healthAPI }}"
+  ASTRIA_BRIDGE_WITHDRAWER_SEQUENCER_COMETBFT_ENDPOINT: "{{ .Values.config.sequencerCometBftEndpoint }}"
```

Comment:
nit: maybe change `sequencerCometBftEndpoint` to `sequencerCometbftEndpoint` to keep consistency with sequencer chart